fs2/shared/src/main/scala/org/atnos/eff/addon/fs2/AsyncTaskInterpreter.scala

```diff
@@ -17,26 +17,49 @@ import scala.concurrent.ExecutionContext
 import scala.concurrent.duration.FiniteDuration
 import scala.util.{Either, Failure, Left, Right, Success}
 
-case class AsyncTaskInterpreter(es: ExecutorServices) extends AsyncInterpreter[Task] { outer =>
+case class AsyncTaskInterpreterEffects()(implicit s: Strategy, sc: Scheduler, es: ExecutorServices) extends AsyncInterpreter[Task] { outer =>
```

Comment:
Yeah much better name.  Well I at least need a ExecutionContext for the future interpreter, but I wasn't sure what granularity to specify.  I could do those 3 individual ones, but figured since most stuff used the `ExecutorServices` it would be easier to keep it around.